samples/highcharts/demo/stellar-chart/demo.css

```diff
@@ -0,0 +1,28 @@
+.highcharts-figure {
+    min-width: 470px;
```

Comment:
We need to find a way to make it work with mobile. Aim for a `min-width` of 360px. I suggest removing the outer, empty grid line (see my other comment), and create a responsive rule for the font size of the center text.

Reply:
Achieved with some font-size adjustments and `responsive.rule`!